deployment/view/onramp.go

```diff
@@ -94,10 +94,14 @@ func fetchDestChainConfig(ctx context.Context, c cldf_ton.Chain, block *ton.Bloc
 	var eg errgroup.Group
```

Comment:
We should probably use the idiomatic `eg, egCtx := errgroup.WithContext(ctx)` so we propagate errors from parallel processes